platforms-serverless-vercel/vercel-with-redwood/web/package.json

```diff
@@ -13,9 +13,11 @@
     ]
   },
   "dependencies": {
+    "@prisma/client": "4.13.0-dev.19",
     "@redwoodjs/forms": "3.8.0",
     "@redwoodjs/router": "3.8.0",
     "@redwoodjs/web": "3.8.0",
+    "prisma": "4.13.0-dev.19",
```

Comment:
This project does not need Prisma. Do we have to install it here?

Reply:
No, that's a mistake.